tests/python_comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -15,7 +15,7 @@ if a:  # a
         # a
     )
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-if a: # a
+if (a): # a
```

Comment:
Seems the comment here is triggering the `ifBreak` node to be printed, even though it's on the same line. Not sure what I'm doing wrong here or if this is just another bug.

Reply:
Hm interesting. Yeah I think when I looked into this the comment has a `hardline` in it, which triggers the `ifBreak` for the line. I'd be okay with leaving this for now and coming back to address it in a later change.